### DISCUSSION TITLE
Add instructions to restore profile if Pin is forgotten

### DISCUSSION THREAD

**Discussion by Phyloiota** [iotaledger/firefly#1540] [Features / Enhancements]
## Description

I fi dont remember my profille Pin i can try 3 times and have to wait 30 seconds, and try again 3 times. This loop is endless continuable without getting solved.  I would suggest to offer an option / button "forgot Pin?- restore Profile from seed vault or menoric phrase here" - leading to a screen that lets me put in my 24 words or upload my seed vault and leads me through the process to get access to the Profile again.

## Motivation

usability is needed. Users otherwise will not know what to do if the Pin is forgotten. They would have to read a troubleshoot doc or ask in the discord.

## Requirements

Write a list of what you want this feature to do.

1.
2.
3.

## Open questions (optional)

Use this section to ask any questions that are related to the feature.

## Are you planning to do it yourself in a pull request?

No.

**Comment by rafaeldjpbrochado** [comment]
You can find a mock-up I did for this in the following threads: https://github.com/iotaledger/firefly/issues/254, https://github.com/iotaledger/firefly/issues/332

![fireflyloginscreen](https://user-images.githubusercontent.com/39466455/111057752-d7a70300-8457-11eb-902e-af425c6c3ead.png)
